src/style.css

```
@@ -0,0 +1,91 @@
+
+@font-face {
+  font-family: Avenir-Book;
+  font-style: normal;
+  font-weight: 300;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-style: normal;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.label {
+  position: relative;
+  width: 100%;
+  display: block;
+  margin-top: 20px;
+}
+
+.big-input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.big-input::placeholder {
+  color: #3d4e61;
+}
+
+.small-input::placeholder {
+  color: #3d4e61;
+}
+
+.img-b {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  top: 45px;
+  padding-left: 26px;
+}
```

Виправ координати для позиції, а також бажано йменувати селектори саме так, що вони значать. До прикладу search-icon, big-search-icon

```suggestion
.img-b {
  position: absolute;
  height: 19px;
  width: 19px;
  top: 25px;
  left: 26px;
}
```